### PR TITLE
Merging new API version from private pr repo https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview

### PR DESCRIPTION
**Pull request by aronip**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by visingla-ms** [comment]
Can you please share Private Repo PR link, which is SignedOff?

**Comment by visingla-ms** [comment]
> ### Swagger Validation Report
> `️️✔️`BreakingChange succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705287532)] [Expand]
> `️⚠️`Breaking Change(Cross-Version): 28 Warnings warning [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705287635)]
> compared swaggers (via Oad [v0.10.2](https://www.npmjs.com/package/@azure/oad/v/0.10.2))]	new version	base version
> connectedvmware.json	2022-07-15-preview([c9bc94f](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json))	2022-01-10-preview([main](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json))
> The following breaking changes are detected by comparison with the latest preview version:
> 
> Rule	Message
> ⚠️ [1008 - ModifiedOperationId](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1008.md)	The operation id has been changed from 'VirtualMachines_Create' to 'VirtualMachines_CreateOrUpdate'. This will impact generated code.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L1828:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L1828:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L1828:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L1828:7)
> ⚠️ [1008 - ModifiedOperationId](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1008.md)	The operation id has been changed from 'VirtualMachines_List' to 'VirtualMachines_ListAll'. This will impact generated code.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2246:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2246:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2237:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2237:7)
> ⚠️ [1008 - ModifiedOperationId](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1008.md)	The operation id has been changed from 'VirtualMachines_ListByResourceGroup' to 'VirtualMachines_List'. This will impact generated code.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2289:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2289:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2280:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2280:7)
> ⚠️ [1008 - ModifiedOperationId](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1008.md)	The operation id has been changed from 'HybridIdentityMetadata_ListByVm' to 'HybridIdentityMetadata_List'. This will impact generated code.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3396:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3396:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3387:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3387:7)
> ⚠️ [1008 - ModifiedOperationId](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1008.md)	The operation id has been changed from 'GuestAgents_ListByVm' to 'GuestAgents_List'. This will impact generated code.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L4001:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L4001:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3933:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3933:7)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L100:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L100:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L157:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L157:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3456:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3456:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3526:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3526:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3593:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3593:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3648:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3648:11)
> ⚠️ [1009 - RemovedRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1009.md)	The required parameter 'name' was removed in the new version.
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3701:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L3701:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L100:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L100:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L157:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L157:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3465:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3465:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3535:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3535:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3602:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3602:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3657:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3657:11)
> ⚠️ [1010 - AddingRequiredParameter](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1010.md)	The required parameter 'virtualMachineName' was added in the new version.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3710:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L3710:11)
> ⚠️ [1017 - ReferenceRedirection](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1017.md)	The '$ref' property points to different models in the old and new versions.
> New: [common-types/resource-management/v3/types.json#L305:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L305:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6150:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6150:9)
> ⚠️ [1017 - ReferenceRedirection](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1017.md)	The '$ref' property points to different models in the old and new versions.
> New: [common-types/resource-management/v3/types.json#L280:11](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L280:11)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6173:11](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6173:11)
> ⚠️ [1025 - RequiredStatusChange](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1025.md)	The 'required' status changed from the old version('True') to the new version('False').
> New: [common-types/resource-management/v3/types.json#L261:7](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L261:7)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6063:7](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6063:7)
> ⚠️ [1029 - ReadonlyPropertyChanged](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1029.md)	The read only property has changed from 'false' to 'true'.
> New: [common-types/resource-management/v3/types.json#L262:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L262:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6064:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6064:9)
> ⚠️ [1029 - ReadonlyPropertyChanged](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1029.md)	The read only property has changed from 'false' to 'true'.
> New: [common-types/resource-management/v3/types.json#L267:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L267:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6068:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6068:9)
> ⚠️ [1029 - ReadonlyPropertyChanged](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1029.md)	The read only property has changed from 'false' to 'true'.
> New: [common-types/resource-management/v3/types.json#L272:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L272:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6072:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6072:9)
> ⚠️ [1029 - ReadonlyPropertyChanged](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1029.md)	The read only property has changed from 'false' to 'true'.
> New: [common-types/resource-management/v3/types.json#L277:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/common-types/resource-management/v3/types.json#L277:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6076:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L6076:9)
> ⚠️ [1042 - ChangedParameterOrder](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1042.md)	The order of parameter 'retain' was changed.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2029:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2029:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2029:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2029:9)
> ⚠️ [1042 - ChangedParameterOrder](https://github.com/Azure/openapi-diff/tree/master/docs/rules/1042.md)	The order of parameter 'force' was changed.
> New: [Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2029:9](https://github.com/Azure/azure-rest-api-specs/blob/c9bc94fe57d90e9d3feb6b9917aea1bcd3cc5558/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json#L2029:9)
> Old: [Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2029:9](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-01-10-preview/connectedvmware.json#L2029:9)
> `️️✔️`CredScan succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705287713)] [Expand]
> `️⚠️`LintDiff: 0 Warnings warning [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705287813)]
> `️️✔️`Avocado succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288008)] [Expand]
> `️️✔️`ApiReadinessCheck succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288121)] [Expand]
> `️⚠️`~[Staging] ServiceAPIReadinessTest: 0 Warnings warning [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288218)]
> `️️✔️`~[Staging] SwaggerAPIView succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288326)] [Expand]
> `️️✔️`ModelValidation succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288420)] [Expand]
> `️️✔️`SemanticValidation succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288500)] [Expand]
> `️️✔️`PoliCheck succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288659)] [Expand]
> `️️✔️`PrettierCheck succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288784)] [Expand]
> `️️✔️`SpellCheck succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705288876)] [Expand]
> `️️✔️`Lint(RPaaS) succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705289015)] [Expand]
> `️️✔️`CadlValidation succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705289104)] [Expand]
> `️️✔️`PR Summary succeeded [[Detail](https://github.com/Azure/azure-rest-api-specs/pull/21658/checks?check_run_id=9705289195)] [Expand]
> Posted by Swagger Pipeline | [How to fix these errors?](https://aka.ms/ci-fix)

Take a look at the breaking changes warnings flagged out in the Breaking Changes section.
Is previous preview version being used by existing customers? and same customers will be using new version also?

**Comment by aronip** [comment]
> Can you please share Private Repo PR link, which is SignedOff?

Here are the list of commits (each commit is a separate PR which was signed off) for this new API version in private repo:
https://github.com/Azure/azure-rest-api-specs-pr/commits/RPSaaSMaster/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview

> Take a look at the breaking changes warnings flagged out in the Breaking Changes section.>
Is previous preview version being used by existing customers? and same customers will be using new version also?

The previous preview version (2022-01-10-preview) is available for customers, but we are still in public preview. The warnings in the new API version seem to be due to a few changes we did in this new API version:
- Updating a few operation names for consistency across multiple Arc RPs (Microsoft.ConnectedVMwarevSphere, Microsoft.Scvmm, Microsoft.AzureStackHCI)
- Moving to common error type from our own definition in the swagger.
- Parameter rename for readability.

We should be ok as we don't have customers with hard dependency on the generated sdks yet.

**Comment by visingla-ms** [comment]
>         "publicKey": {

Why is this property removed?
Also, this property was added in new version?

---
Refers to: specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json:5010 in 63e2787. [](commit_id = 63e27877973a484c1795964dc911ec1ed8460159, deletion_comment = True)

**Comment by visingla-ms** [comment]
Rest LGTM.

**Comment by aronip** [comment]
> > ```
> >     "publicKey": {
> > ```
> 
> Why is this property removed? Also, this property was added in new version?
> 
> Refers to: specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json:5010 in [63e2787](https://github.com/Azure/azure-rest-api-specs/commit/63e27877973a484c1795964dc911ec1ed8460159). [](commit_id = [63e2787](https://github.com/Azure/azure-rest-api-specs/commit/63e27877973a484c1795964dc911ec1ed8460159), deletion_comment = True)

Yes, during the development of the new API version, we added top-level publicKey field in VMProperties. Later we decided to move it to guestAgentProfile.clientPublicKey as that fits better in our scenario. This is not a breaking change as this version is not yet available to customers.

File: specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json [hunk 1]
```diff
@@ -0,0 +1,7500 @@
...
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Request payload.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
```
Comment (ArcturusZhang) [inline specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2022-07-15-preview/connectedvmware.json:565]: Should this body paraemter be required?

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**